packages/tsconfig/package.json

```diff
@@ -5,7 +5,6 @@
   "main": "index.js",
   "files": [
     "base.json",
-    "nextjs.json",
```

Comment:
This has been used by one of the apps created as part of the turborepo boilerplate that I removed